src/app.ts

```diff
@@ -688,6 +716,133 @@ export class App {
     this.writeTests(destinationDirectory, api);
   }
 
+  private writePropertyValue(scope: TypescriptTextWriter, api: gapi.client.discovery.RestDescription, property: gapi.client.discovery.JsonSchema) {
+    switch (property.type) {
```

Comment:
You might also want to check for `"repeated": true` which means that property might be a single value, or array of type. But since it's optional, I don't know what to do about it. Maybe we'll add another test, to test both single value and array later. Other than that, LGTM.